### PR TITLE
Document a better pattern for accessing the API

### PR DESCRIPTION
**Pull request by ijdickinson**
epimorphics/hmlr-linked-data#84

Document a better way to access the API during development than running
an ssh tunnel.

Note that I'm not sure how to get the identity of the SapiNT Docker container,
(other than "Ask Dave" :)), so if someone can suggest a better way that
would be great.

### CODE REVIEW

**Review by joescottdave — APPROVED** [review]
Really should try this out some time next week.  Looks understandable enough in these terms.

File: README.md [hunk 1]
```diff
@@ -21,10 +21,51 @@ definition_ (DSD), to convert the DSL queries to SPARQL.
...
+ :: Spring Boot ::        (v2.2.0.RELEASE)
+
+{"ts":"2022-03-21T16:12:26.585Z","version":"1","logger_name":"com.epimorphics.SapiNtApplicationKt",
+"thread_name":"main","level":"INFO","level_value":20000,
+"message":"No active profile set, falling back to default profiles: default"}
+```
+
+Note that the identity of the Docker image will change periodically. There's
+probably a smart way to find out which is the latest version to run, but I'm
+not sure what it is.
```
Comment (der) [inline README.md]: The currently deployed dev api image version is given by the `api` tag in the ansible [dev configuration](https://github.com/epimorphics/hmlr-ansible-deployment/blob/master/ansible/group_vars/dev/tags.yml).

If you need to run a different API version then the easiest route to discovering the most recent is to use the [AWS ECR console](https://eu-west-1.console.aws.amazon.com/ecr/repositories/private/018852084843/epimorphics/lr-data-api/dev?region=eu-west-1) or look at the hash to the relevant commit in [lr-data-api](https://github.com/epimorphics/lr-data-api).

The

**Review by der — APPROVED** [review]
lgtm

File: README.md (hunk 1 above)
Comment (ijdickinson) [reply README.md]: Thanks, added

**Pull request merged**